cutplace/interface.py

```diff
@@ -524,9 +523,28 @@ def as_sql_create_table(self, dialect='ansi'):
             if len(constraint) > 0:
                 constraints += constraint + ",\n"
 
+        constraints = constraints.rsplit(',', 1)[0]
+
         create_table += constraints
 
-        create_table += ");"
+        create_table += "\n);"
+
+        con = None
+
+        try:
+            con = lite.connect("database.db")
+            cur = con.cursor()
+            cur.execute(create_table)
+
+        except lite.Error as err:
+            return err
+
+        finally:
+            if con:
+                cur = con.cursor()
+                cur.execute("drop table "+table_name[0]+" ;")
```

Comment:
It seems that currently `ant pep8` detects PEP8 issues but does not report them. Try this in the console:
```
pip install -r requirements.txt
pip install -r requirements-dev.txt
```
Then check for PEP8 issues:
```
tox -e flake8
```